### PR TITLE
Internal refactor

### PR DESCRIPTION
**Pull request by paylakatel**
This PR:
- makes the Map component and the Filters component siblings
- tries to make the map extend to the entire viewport and not be a static number of pixels
- implements setMapEl method to load the map

### CODE REVIEW

File: components/Filters.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+              id="from"
+              type="date"
+              onChange={props.fromChange}
+              name="from"
+              value={props.fromDate}
+              className="mb-1"
+            />
+            <Label for="to">To Date:</Label>
+            <Input
+              id="from"
```
Comment (fionawhim) [inline components/Filters.js]: `id="to"`

File: components/Layout.js [hunk 2]
```diff
@@ -25,13 +26,20 @@ export default class Layout extends React.Component {
             content="width=device-width, initial-scale=1.0"
           />
         </Head>
+
+        {/* Make sure the heigh of the body element is the entire view port */}
         <style global jsx>{`
           @import url('https://fonts.googleapis.com/css?family=Roboto:400,600');
           body {
             font-family: 'Roboto';
+            min-height: 100vh;
```
Comment (fionawhim) [inline components/Layout.js:35]: 👍

File: components/MapContainer.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  makeFeaturesQuery = (modeSelection, fromDate, toDate) => {
+    // set query for when "all" modes are selected
+    const allModesSelected = `dispatch_ts >=
+      '${fromDate}' AND 
+      dispatch_ts <= '${toDate}'`;
+
+    // set query for when one mode is selected
+    const oneModeSelected = `mode_type =
+      '${modeSelection}' AND 
+      dispatch_ts >= '${fromDate}'AND
```
Comment (fionawhim) [inline components/MapContainer.js]: Space before `AND`?

**Review by fionawhim — APPROVED** [review]
LGTM after a few tweaks. Great work!

**Comment by paylakatel** [comment]
Thank you!!

**Pull request merged**